diff --git a/dist/checkers/eslint/main.js b/dist/checkers/eslint/main.js
index 27a775515049aa60f23c2778632b86d2db6d3513..58983773e289e117b4adffc038807eff6d631206 100644
--- a/dist/checkers/eslint/main.js
+++ b/dist/checkers/eslint/main.js
@@ -66,7 +66,12 @@ const createDiagnostic = (pluginConfig) => {
         } = require2("eslint/use-at-your-own-risk");
         if (shouldUseFlatConfig == null ? void 0 : shouldUseFlatConfig()) {
           eslint = new FlatESLint({
-            cwd: root
+            cwd: root,
+            cache: eslintOptions.cache,
+            cacheLocation: eslintOptions.cacheLocation,
+            cacheStrategy: eslintOptions.cacheStrategy,        
+            overrideConfigFile: eslintOptions.overrideConfigFile,
+            errorOnUnmatchedPattern: eslintOptions.errorOnUnmatchedPattern,
           });
         } else {
           throw Error(
