diff --git a/dist/checkers/eslint/main.js b/dist/checkers/eslint/main.js
index 5f44bab2c5753726fe9f8d877a519c428e479dcb..bd7baaef575ad75281bf3aed0144a79c8ff04524 100644
--- a/dist/checkers/eslint/main.js
+++ b/dist/checkers/eslint/main.js
@@ -66,7 +66,12 @@ const createDiagnostic = (pluginConfig) => {
         } = require2("eslint/use-at-your-own-risk");
         if (shouldUseFlatConfig == null ? void 0 : shouldUseFlatConfig()) {
           eslint = new FlatESLint({
-            cwd: root
+            cwd: root,
+            cache: eslintOptions.cache,
+            cacheLocation: eslintOptions.cacheLocation,
+            cacheStrategy: eslintOptions.cacheStrategy,        
+            overrideConfigFile: eslintOptions.overrideConfigFile,
+            errorOnUnmatchedPattern: eslintOptions.errorOnUnmatchedPattern,
           });
         } else {
           throw Error(
